### PR TITLE
fix: hidden context variable in UI schema

### PR DESCRIPTION
**Pull request by alexcodelf**
**Problem:**

UI schema should not show context. Walrus will auto inject context when deploying, user does not care about this field.

**Solution:**

Since we need keep it in schema of the template to determine whether inject walrus context to the template. **The proposed solution involves excluding the context field when generating the UI schema file for the template. If users wish to modify the UI schema in Walrus, the context field will not be hidden, allowing for customization.**

**Related Issue:**
#1528

### CODE REVIEW

**Comment by gitlawr** [comment]
What happen when users edit and save a custom UI schema

**Comment by alexcodelf** [comment]
> What happen when users edit and save a custom UI schema

Users will not see the context field in `UIshcema` and rendered form，the deployment will run normally as we using `schema` to determine whether to inject context

File: pkg/templates/git.go [hunk 1]
```diff
@@ -158,7 +163,8 @@ func syncTemplateFromRef(
 			Name:       entity.Name,
 			Version:    ref,
 			Source:     source + "?ref=" + repo.Reference,
-			Schema:     *schema,
+			Schema:     *originSchema,
+			UiSchema:   fileSchema.Expose(),
```
Comment (gitlawr) [inline pkg/templates/git.go:167]: What happen when we revert uiSchema to the default?

Comment (alexcodelf) [reply pkg/templates/git.go:154]: The reset logic before will reset all template version and UI schema. Using template version bus to revert to default.

**Review by aiwantaozi — APPROVED** [review]

**Review by gitlawr — APPROVED** [review]
LGTM

**Pull request merged**